### ISSUE TITLE
PyTorch 深度学习

### ISSUE THREAD

**Issue by boy-hack** [boy-hack/boy-hack#29]
1. tensor 张量 
    - 0阶 1阶 2阶  
2. tensor的方法
    - t1 = tensor([1,2])
    - t1.size()
    - t1.item()
    - t1.numpy() # 转python类型
    - t1.view(-1) # 转成一维向量
    - t1.dim() # 获取阶数
3. tensor的切片
    - t2 = torch.rand(3,4,2) #三维
    - t2[1,:,:]

**Comment by boy-hack** [comment]
1. 使用GPU加载
    - print(torch.cuda_is_available())
    - device1 = torch.device('cuda')
    - device2 = torch.device('cpu')
    - label = torch.tensor([1,2,3])
    - label.to(device1)
    - label.to(device2)
